svc-lighthouse-api/src/test/java/gov/va/vro/abddataaccess/service/FhirClientTest.java

```diff
@@ -178,6 +178,10 @@ public void testGetBloodPressure() throws AbdException {
     mockRest(conditionResp, "Condition");
     AbdEvidence evidence = client.getMedicalEvidence(testClaim);
     assertNotNull(evidence);
+    //
+    // TODO: It seems using bpBundleInfo has some problems with
+    // this test. We need to look into it later.
```

Comment:
Do we still need this comment. Looks like you put it back?

Reply:
Yes, if we want to fix problems in FhirClientTest.